okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -243,6 +245,30 @@ public final OkHttpChannelBuilder sslSocketFactory(SSLSocketFactory factory) {
   }
 
   /**
+   * Set the hostname verifier to use when using TLS negotiation. The hostnameVerifier is only used
+   * if an sslSocketFactory is set. If the hostname verifier is not set, a default hostname
```

Comment:
Oops, right you are; I missed that `createSocketFactory` makes one for you. I changed it to say that it's only used when using TLS negotiation, which I think is accurate.